deploy/helm/kubecf/templates/_capi.tpl

```diff
@@ -0,0 +1,31 @@
+{{- /*
+==========================================================================================
+| _capi.setProperty $property $value
++-----------------------------------------------------------------------------------------
+| There are 4 CAPI jobs that all share the same cloud_controller_ng specs file,
+| and therefore potentially all use the same "cc" properties. This template will
+| set a property in all of the jobs, so there is a single location to keep track
+| of instance groups these jobs run in.
+|
+| $property can use dotted path notation to specify nested properties,
+| e.g. "diego.foo" to set the "cc.diego.foo" property.
+==========================================================================================
+*/}}
+{{- define "_capi.setProperty" }}
+  {{- $property := index . 0 }}
+  {{- $value := index . 1 }}
+
+  {{- $ig := dict }}
```

Comment:
Hey, it is not my fault that Go templates use the `$` sigil for maps and slices too. It would be obvious if they went all the way in copying Perl, so I could have called it `%ig`. 😛